DotNet-Platform.md

```diff
@@ -0,0 +1,40 @@
+![Logo][logo]
+
+# The .NET Platform 
+
+**.NET platform** is big and comprehensive collection of frameworks and libraries manager by [Microsoft](https://www.microsoft.com/net). In the beginning it could be quite daunting to get familiar with .Net ecosystem.
+
+Our aim is to get you familiar with the .Net platform, so that you are ready to learn the [c# programing language](C-Sharp) along with development environment, project structures, techniques and best practices you will be performing as a professional developer.
+
+Let's get some of the jargon's out of the way, don't worry, you don't need to know in-depth about these at this point.
+
+* [.Net Framework](https://msdn.microsoft.com/en-gb/library/w0x726c2(v=vs.110).aspx) : (_pronounced dot net_) provides configuration for your application to run on top of it. 
+It provides all the services an app needs for instance communication with database, networks, file systems which are commonly used to build console, desktop, web, mobile and game applications.
+
+ * CLR (Common Language Runtime) : is an execution environment for your application, which manages it's life-cycle.
+     1. It provides services like memory management.
+     2. Is operating system and hardware independent.
+     3. and language independent, for instance can be used for c#, vb.net and f#.
+ 
+ * C-Sharp Compiler : a csc.exe is a compiler, which transforms c# code into Microsoft Intermediate Language (MSIL), which is commonly referred as IL. It defines instructions which CLR can understand. CLR job is to read these instructions and convert into instructions which your machine's CPU undertansds.
+ 
+ * Class libraries : It containes thousands of in-build classes for your application use, for example System.IO for reading/writing data streams, HttpClient send data over network, ASP.NET for web applications, ADO.NET for data access to relational databases (like Microsoft SQL Server and MySQL), and Windows Communication Foundation (WCF) for service-oriented applications which communicates over established protocols like like HTTP, REST, SOAP and TCP etc.
+
+ * The C# Programming Language (_pronounced "C-sharp"_) : 
+    1. used to write applications, services and reusable libraries.
+    2. Has syntax similar as Java, c++ and javascript. 
```

Comment:
> Has syntax similar as Java, c++ and javascript. 

Why are you capitalizing "H" at the beginning? It's part of a sentence here.

Also, use proper nouns like language names with proper case.

>~~c++~~ C++

> ~~javascript~~ JavaScript